packages/terra-site/src/App.jsx

```diff
@@ -22,26 +21,39 @@ const propTypes = {
 
 const locale = document.getElementsByTagName('html')[0].getAttribute('lang');
 
+const themes = {
+  'Defualt Theme': '',
+  'Consumer Theme': 'cerner-consumer-theme',
+  'Mock Theme': 'cerner-mock-theme',
+};
+
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
       isOpen: window.innerWidth >= 768,
+      isLtr: true,
```

Comment:
This is completely up to preference. An alternative to `isLtr` would be `dir`. Which would read "The state of directionality is left to right / right to left" instead of "The state of left to right is true / false"